Twig/TwigExtension.php

```diff
@@ -51,6 +60,31 @@ public function getFunctions()
         ];
     }
 
+    /**
+     * Formats a float value as an amount (money).
+     *
+     * @param $number
+     * @return string
+     */
+    public function getFormatAmount($number)
+    {
+        return number_format($number, 2, ',', '.');
+    }
+
+    /**
+     * Formats a float value as a percent string value, with an optional unit in the end.
+     *
+     * @param $number
+     * @param bool $includeUnit
+     * @return string
+     */
+    public function getFormatPercent($number, $includeUnit = true)
+    {
+        $number = number_format($number * 100.0, 2, ',', '.');
+        $number = preg_replace("/\,?0+$/", "", $number);
```

Comment:
What is going on here?! Why not use `0` (rather than `2`) when calling `number_format`?

Reply:
Added a comment.
It is just to allow 2 decimals after the zero, if they exist.

Reply:
But why do we want to remove trailing zeros after the comma?
The regexp could be just `/0+$/` since we know that any trailing zeros come after the comma (we always have two digits after the comma).
Oh, and btw, strings should be enclosed in `'`.
We could also use https://twig-extensions.readthedocs.io/en/latest/intl.html – but let's move on to more important issues …

Reply:
Point taken, changed to twig/extensions filters instead